CHANGELOG.md

```diff
@@ -28,7 +28,7 @@
 ## Fixes
 
 - Fixed an issue where some types on the `Call` and `Voice` classes were being incorrectly exported. Types and references to `addEventListener` are instead now correctly exported as `addListener`.
-- Fixed an issue where available audio devices were incorrectly emitted and returned by the SDK.
+- Fixed an issue where available audio devices were sometimes incorrectly emitted and returned by the SDK on Android platforms. This occurs more frequently in development environments when the JS bundle is reloaded, but could happen in production environments as well.
```

Comment:
@mhuynh5757 - how do you force metro bundler to reload in production environments?

Reply:
This statement is more speculation, good point. It's not "forcing Metro bundler to reload" in prod environments, since the JS is already built and bundled, but I'm not sure if that means the native application will never reload the JS bundle.